pool_automation/roles/aws_manage/defaults/main.yml

```diff
@@ -30,3 +24,12 @@ aws_regions:
   - us-east-2
   - us-west-1
   - us-west-2
+
+instance_count: 4
+
+pa_aws_prefix: PA
```

Comment:
AFAIK this is going to add another tag to EC2 instances, so probably this better be named as `aws_tag_prefix` to be consistent with `aws_tag_namespace` and `aws_tag_role`. Also `prefix` looks very general here, probably something like `owner` or `dept` or `purpose` would be better.

Reply:
Yes, it is going but not in that PR and it would be likely something like `Project`. It wouldn't be a tag prefix but rather a separate tag itself. Here it is used only for resources where no tags are created (for now) and it's really just a prefix for names.